docs/00-project/ai/mcp-governance.md

```
@@ -58,47 +58,52 @@ generated MCP configuration.
 - **Описание:** Auto-recall настроен в `.codex/AGENTS.md`
 - **Настройка:** `DEJA_AUTO_RECALL_PATH` указывает на `.codex/AGENTS.md`
 - **Обёртка:** `scripts/ai/mcp/mcp_deja_wrapper.sh`
+- **Windows companion:** `scripts/ai/mcp/mcp_deja_wrapper.ps1`
 
 ### adr-analysis
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)